dataactvalidator/config/sqlrules/c11_cross_file.sql

```diff
@@ -0,0 +1,13 @@
+SELECT
+	af.row_number,
+	af.piid,
+    af.parent_award_id
+FROM award_financial AS af
+WHERE af.submission_id = {}
+	AND NOT EXISTS (
+		SELECT 1
+		FROM award_procurement AS ap
+		WHERE ap.submission_id = af.submission_id
+		    AND ap.piid = af.piid
+		    AND ap.parent_award_id = af.parent_award_id
```

Comment:
After talking to Kaitlin (if i remember correctly), same reporting period is satisfied by making sure the submission IDs match, since the submission is what records the reporting period.